packages/sit-onyx/src/components/OnyxLink/types.ts

```diff
@@ -8,6 +8,11 @@ export type OnyxLinkProps = {
    * For `_blank`, the `rel="noopener noreferrer"` will be set automatically.
    */
   target?: LinkTarget;
+  /**
+   * Whether to hide the external link icon if `href` points to an external website.
+   * For links not starting with "http://" or "https://" it will be hidden automatically.
+   */
+  hideExternalIcon?: boolean;
```

Comment:
It might be a nicer API to offer the option to set it directly:
```suggestion
hasExternalIcon?: boolean;
```

And per default, when it is `undefined`, we use our logic to determine the external icon

Reply:
Good idea! I changed to type to `boolean | "auto"` so its clearer to the user that we use our logic